nym-vpn-apple/Home/Sources/Home/HomeViewModel.swift

```diff
@@ -352,6 +352,7 @@ private extension HomeViewModel {
     }
 
     func needsHelperUpdate() async -> Bool {
+        guard helperManager.isHelperAuthorizedAndRunning() else { return false }
```

Comment:
So what happens if helper tool is out of date but not running?

Reply:
It will get reinstalled.